### PR TITLE
Lint all files

### PR DESCRIPTION
**Pull request by kcpevey**
Somehow the files in this repo have gotten past the linter.

I've run `yarn run format:fix` in the `/docs/docs` directory and this PR is what was modified. 

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

**Comment by viniciusdc** [comment]
I need to have a look in the built docs, I am concerned with the tables strucutures

**Comment by kcpevey** [comment]
@viniciusdc I went back through and confirmed that the tables all still look good after the linting.

File: docs/docs/how-tos/nebari-environment-management.md
Comment (viniciusdc) [inline docs/docs/how-tos/nebari-environment-management.md:13]:
```suggestion
This is a simple way to set up a new Conda environment named `nebari0` with a specific Nebari version and exactly one extension installed. This `nebari0` environment will store all of our packages, including those installed with `pip` because when **pip** is installed into an active conda environment, packages which are then installed with pip will not be saved in the global namespace_. This especially critical when working with extensions, because `nebari` will detect and apply any extensions installed in your Python environment when it runs.
```

Comment (kcpevey) [reply docs/docs/how-tos/nebari-environment-management.md:13]: Now that I'm actually reading what this says... its completely wrong. We just need to delete this file altogether. I can open a separate PR for that.

**Comment by viniciusdc** [comment]
@kcpevey  is this ready?

**Comment by kcpevey** [comment]
@viniciusdc yes, just waiting on approval

**Review by viniciusdc — APPROVED** [review]
LGTM, thanks @kcpevey

**Pull request merged**
